sdk/storage/azure-storage-blob/samples/blob_samples_containers.py

```diff
@@ -44,7 +45,7 @@ def container_sample(self):
         # [START create_container_client_sasurl]
         from azure.storage.blob import ContainerClient
 
-        sas_url = sas_url = "https://account.blob.core.windows.net/mycontainer?sv=2015-04-05&st=2015-04-29T22%3A18%3A26Z&se=2015-04-30T02%3A23%3A26Z&sr=b&sp=rw&sip=168.1.5.60-168.1.5.70&spr=https&sig=Z%2FRHIX5Xcg0Mq2rqI3OlWTjEg2tYkboXr1P9ZUXDtkk%3D"
+        sas_url = "https://account.blob.core.windows.net/mycontainer?sv=2015-04-05&st=2015-04-29T22%3A18%3A26Z&se=2015-04-30T02%3A23%3A26Z&sr=b&sp=rw&sip=168.1.5.60-168.1.5.70&spr=https&sig=Z%2FRHIX5Xcg0Mq2rqI3OlWTjEg2tYkboXr1P9ZUXDtkk%3D"
```

Comment:
Although, I'm certain these are fake values, do you think it;s a good idea for us to generate sas url / import from env variable in the samples?

Reply:
I think the fake one is pretty straight forward. ^_^
Also when this code snippet is to tell user how to instantiate client using sas url directly